compiler-rt/cmake/config-ix.cmake

```diff
@@ -457,25 +457,32 @@ if(APPLE)
 
   # Note: In order to target x86_64h on OS X the minimum deployment target must
   # be 10.8 or higher.
+  set(SANITIZER_MIN_OSX_VERSION "" CACHE STRING
+    "Minimum OS X version to target (e.g. 10.10) for sanitizers.")
```

Comment:
Could we reverse the order, that is compute the default value in the block below, and then set the cached value as
```cmake
set(SANITIZER_MIN_OSX_VERSION "${MIN_OSX_VERSION}" CACHE STRING
    "Minimum OS X version to target (e.g. 10.10) for sanitizers.")
```
after the block?

Reply:
Addressed!